.github/workflows/cmake.yml

```diff
@@ -1,7 +1,16 @@
 name: Build, lint, and test
 
-on: [push, pull_request]
-
+on:
+  pull_request:
+    paths-ignore:
+      - "**/azure-pipelines.yml"
+      - "**/README.md"
```

Comment:
This could probably include a couple more files: 
- `SECURITY.md`
- `CONTRIBUTING.md`
- `.mailmap`

Reply:
Potentially including all `.md` files could be a nice set